### ISSUE TITLE
[Qt-Wallet] Notifications prevent button usage in some windows.

### ISSUE THREAD

**Issue by scasaletto** [mwcproject/mwc-qt-wallet#540]
Need to review how usable each window is when notifications are being displayed. K says HODL window is not very usable when he gets a lot of notifications coming in due to transactions. Review each window in the wallet and determine whether decreasing the size of the status window will help or need to decrease the number of windows displayed at any one time.

**Comment by scasaletto** [comment]
Resized the notification windows so that they are not so large.
Added a new wallet config option to enable/disable the new notificaiton windows. When the notification windows are disabled, the message box that used to pop-up when you receive MWC will now pop up again.
In the wallet config window, I also centered the Reset/Apply buttons so the notification windows don't overlap them so much that they cannot be selected.
The main screens that will still have an issue are the Transaction and Outputs screens. If a lot of notifications are constantly coming in, the buttons to go to the next/prev set of transactions or outputs will be covered. There is not much I can do about that. So the new config option to disable the notifications will at least help people who are annoyed by this behavior.
